syncd/syncd.cpp

```diff
@@ -40,6 +40,7 @@ void exit_and_notify(int status)
     exit(status);
 }
 
+#ifdef BRCMSAI
```

Comment:
SAI_SWITCH_ATTR_CUSTOM_RANGE_BASE  was changed to SAI_SWITCH_ATTR_CUSTOM_RANGE_START 

simple change could satisfy both: and later on when sai will be stable it can be changed back

int32_t id = -1;

#ifndef SAI_SWITCH_ATTR_CUSTOM_RANGE_BASE 
id = SAI_SWITCH_ATTR_CUSTOM_RANGE_BASE;
#elsif  SAI_SWITCH_ATTR_CUSTOM_RANGE_START
id = SAI_SWITCH_ATTR_CUSTOM_RANGE_START;  
#else
#error invalid atrib custom range not defined
#endif